mmv1/third_party/terraform/tests/resource_eventarc_trigger_channel_test.go.erb

```diff
@@ -0,0 +1,178 @@
+<% autogen_exception -%>
```

Comment:
nit: By convention, we'll want to name this "resource_eventarc_trigger_test.go" (removing "channel")

Reply:
Done